src/Components/events/NewEvent.js

```diff
@@ -76,24 +70,25 @@ const NewEvent = () => {
       });
       return;
     } else {
-      const informationUser = getFromLocalStorage();
-      const idUser = informationUser.idUser;
-      let host = `${informationUser.name} ${informationUser.lastName}`;
-
-      eventsList.push({
-        id: uuidv4(),
-        date: dateToSave,
-        time: timeEvent,
-        nameEvent: titleEvent,
-        host: host,
-        descriptionEvent: descriptionEvent,
-        attendees: 1,
-        capacity: capacityPeopleEvent,
-        stateEvent: stateEvent,
-        users: [idUser],
-      });
-
-      localStorage.setItem("Events", JSON.stringify(eventsList));
+      const userInformation = getFromLocalStorage();
+      const idUser = userInformation.idUser;
+      let host = `${userInformation.name} ${userInformation.lastName}`;
+      try {
+        await saveEvent({
+          id: uuidv4(),
```

Comment:
add another property called `eventOwner` and set the idUser as value, eventOwner: idUser